functions/effect_increment.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FUNCTIONS_EFFECT_INCREMENT_H
+#define FUNCTIONS_EFFECT_INCREMENT_H
+
+// Usage: EffectIncrement<EFFECT, MAX, I>
+// Increases by value I (up to MAX) each time EFFECT is triggered
+// if greater than MAX returns 0
+// I, MAX = numbers
+// return value: INTEGER
+
+template<BladeEffectType EFFECT, class MAX = Int<32768>, class I = Int<1>>
+class EffectIncrement {
```

Comment:
Actually, can we name this EffectIncrementF to make it obvious that it's a function?